README.md

```diff
@@ -1,3 +1,16 @@
 ## Mujoco Robot Arm
 
 Experiments to learn Mujoco and setting up traditional + RL controllers for a robotic arm environment within in. 
+
+Required Libraries: 
+- NumPy
+- Mujoco
+- MatplotLib 
+
+Next, run the example, 
+`mjpython main_IK.py` 
```

Comment:
this should just `python3 main_IK.py`, right?

Actually, I searched it and it seems like this is a macOS specific thing per the warning here? https://mujoco.readthedocs.io/en/stable/python.html#passive-viewer